packages/svelte-hmr/package.json

```
@@ -23,7 +23,7 @@
     "node": "^12.20 || ^14.13.1 || >= 16"
   },
   "peerDependencies": {
-    "svelte": ">=3.19.0"
+    "svelte": "^3.19.0 || ^4.0.0-next.0"
```

So we don't need to do this again after the final release 
```suggestion
    "svelte": "^3.19.0 || ^4.0.0-next.0 || ^4.0.0"
```